src/drivers/intel/hda/hda.c

```diff
@@ -32,6 +32,10 @@ static int hda_set_config(struct dai *dai,  struct ipc_config_dai *common_config
 	struct sof_ipc_dai_config *dai_config = spec_config;
 	struct sof_ipc_dai_hda_params *params = &dai_config->hda;
 
+	/* no params setting in blob */
```

Comment:
what is used in the blob for lp mode. Can we state this in the comment.

Reply:
sure